src/main/resources/data/minecraft/tags/blocks/natural_stone.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Is this actually used? It looks like the `CottonOreFeature` code is still using a predicate. Also, why is it in the `minecraft` namespace?

Reply:
Well that tag is meant to replace the predicates, however the code only checks for replacement specs if they are not null (by default all ores have an end and nether spec eince they have nether ore variants) so the stone variants will never work. Hence why I had to add a spec for the stone based ores which use this tag. I could definitely migrate this to the Cotton namespace. So I could get rid of that basically unused code.

Reply:
Okay I moved this to the Common namespace.